Library/Formula/cpp-netlib.rb

```diff
@@ -5,6 +5,12 @@ class CppNetlib < Formula
   version "0.11.2"
   sha256 "71953379c5a6fab618cbda9ac6639d87b35cab0600a4450a7392bc08c930f2b1"
 
+  devel do
+    url "https://github.com/cpp-netlib/cpp-netlib/archive/cpp-netlib-0.12.0-rc1.tar.gz"
+    version "0.12.0"
```

Comment:
Should this be `"0.12.0-rc1"`, to differentiate it from the eventual production release, and allow automatic `upgrade`s if an `rc2` comes out?